routes/songlist-db.js

```diff
@@ -0,0 +1,145 @@
+var express = require('express'),
+    router = express.Router(),
+    Song = require('../models/song'),
+    User = require('../models/user'),
+    helperFunc = require('../lib/passport')
+
+
+// router.use('/', isAdminLoggedIn, function(req, res, next, next){
+//   next()
+// })
+
+router.get('/', function(req, res, next) {
+    Song.find(function(err, songs) {
+        if (err) return next(err);
+        res.render('songlist-db', {
+            songs: songs
+        })
+    })
+})
+
+router.get('/add', function(req, res, next) {
+    res.render('add', {
```

Comment:
if you're going to use defaults, specify them via db or config or something instead of putting it in the code. This will be problematic if we ever need to change it since it could be in many places.